code/modules/mob/living/carbon/human/species/unathi.dm

```diff
@@ -57,49 +57,46 @@
 		"is twisting their own neck!",
 		"is holding their breath!")
 
-
 /datum/species/unathi/on_species_gain(mob/living/carbon/human/H)
 	..()
-	var/datum/action/innate/tail_lash/lash = new()
-	lash.Grant(H)
+	var/datum/action/innate/ignite/fire = new()
+	fire.Grant(H)
 
 /datum/species/unathi/on_species_loss(mob/living/carbon/human/H)
 	..()
-	for(var/datum/action/innate/tail_lash/lash in H.actions)
-		lash.Remove(H)
-
-/datum/action/innate/tail_lash
-	name = "Tail lash"
-	icon_icon = 'icons/effects/effects.dmi'
-	button_icon_state = "tail"
-	check_flags = AB_CHECK_LYING | AB_CHECK_CONSCIOUS | AB_CHECK_STUNNED
-
-/datum/action/innate/tail_lash/Activate()
+	for(var/datum/action/innate/ignite/fire in H.actions)
+		fire.Remove(H)
+
+/datum/action/innate/ignite
+	name = "Ignite"
+	desc = "You form fire in your mouth, allowing you to.. light cigarettes. Requires you to drink welding fuel beforehand."
+	icon_icon = 'icons/obj/cigarettes.dmi'
+	button_icon_state = "match_unathi"
+	var/cooldown = 0
+	check_flags = AB_CHECK_HANDS_BLOCKED | AB_CHECK_CONSCIOUS | AB_CHECK_STUNNED
```

Comment:
```suggestion
	check_flags = AB_CHECK_HANDS_BLOCKED
```
if they are unconscious or stunned their hands are blocked. so the others are redundant